### ISSUE TITLE
1.5 destroyed Item Details

### ISSUE THREAD

**Issue by bukakow** [reupen/columns_ui#337]
![item_details](https://user-images.githubusercontent.com/36563739/92135753-73054e80-ee0b-11ea-8317-748d493f8b20.png)

**Comment by reupen** [comment]
Hi,

Thanks for the report. Could you share an exported Columns UI configuration (FCL file) or details of the Items details configuration you are using?

Could you also confirm what OS you are running?

Thanks

**Comment by bukakow** [comment]
The Item Details script is long, non-readable and related to many plugins, so I did not include it
Windows 10.

[stuff.zip](https://github.com/reupen/columns_ui/files/5169995/stuff.zip)

**Comment by reupen** [comment]
Please try the version in this archive: [Test version](https://artprodcus3.artifacts.visualstudio.com/Ac14dc90d-58e8-48df-a92d-d34cda9b6d7e/b518cdbd-8215-43ba-adb7-db580f6fd944/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3JldXBlbi9wcm9qZWN0SWQvYjUxOGNkYmQtODIxNS00M2JhLWFkYjctZGI1ODBmNmZkOTQ0L2J1aWxkSWQvOTIxL2FydGlmYWN0TmFtZS9WUysyMDE5K3YxNDIrUmVsZWFzZQ2/content?format=zip)

In that archive, within `VS 2019 v142 Release\vc16\Release\` you'll find an .fb2k-component file that you can install in the usual way (you can ignore the .pdb file).

There are still some differences regarding line height, but the cases I checked were bugs in the older version.

For example, where you have:
```
/// -------------------------------------------------------------------------
///                 TRACK INFORMATIONS (KBPS, DR, HZ...)
/// -------------------------------------------------------------------------
$set_font(Segoe UI,7;)
$crlf()
[%bitrate% kbps]
```

The old version was making the height of the bitrate line 14pt (the font prior to that `$set_font()` call) instead of 7pt.

To achieve similar behaviour to the old version, you can do any of the following:

1. Force a taller line height by using another `$set_font()`:

    ```
    /// -------------------------------------------------------------------------
    ///                 TRACK INFORMATIONS (KBPS, DR, HZ...)
    /// -------------------------------------------------------------------------

    $crlf()
    $set_font(Segoe UI,14;)
    $set_font(Segoe UI,7;)
    [%bitrate% kbps]
    ```

2. Use a zero-width space to carry forward the previous font:

    ```
    /// -------------------------------------------------------------------------
    ///                 TRACK INFORMATIONS (KBPS, DR, HZ...)
    /// -------------------------------------------------------------------------

    $crlf()
    // Zero-width space
    $char(8203) 
    $set_font(Segoe UI,7;)
    [%bitrate% kbps]
    ```

3. Use two blank lines (this is slightly taller due to padding between the lines):

    ```
    /// -------------------------------------------------------------------------
    ///                 TRACK INFORMATIONS (KBPS, DR, HZ...)
    /// -------------------------------------------------------------------------

    $crlf()
    $crlf()
    $set_font(Segoe UI,7;)
    [%bitrate% kbps]
    ```

**Comment by bukakow** [comment]
Works great.

**Comment by reupen** [comment]
Great, thanks. This will be released as 1.6.0 in the next day or so.
